packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -995,6 +1002,10 @@ module.exports = (
           uid = auth.credentials.uid;
           scope = { contains: () => true };
           client_id = null;
+        } else if (auth.strategy === 'profileSecret') {
```

Comment:
Replace `profileSecret` with `subscriptionSecret` and with that, hard code the `profile:subscriptions` scope.